tests/integration/test_assets.py

```diff
@@ -102,6 +101,18 @@ def test_full_flow():
     assert removed
 
 
+def name_exists_for_some_some_transaction(report):
+    i = 0
+    while i < len(report['items'][0]['accounts']):
+        if len(report['items'][0]['accounts'][i]['transactions']) > 0:
+            return (report['items'][0]['accounts'][i]
+                    ['transactions'][0]['name']
+                    is not None)
+        i = i + 1
+
```

Comment:
Sorry for style-comment, can we please use a for-each loop here?

Reply:
⚡️